css/src/components/gradient-card.scss

```diff
@@ -0,0 +1,11 @@
+.gradient-card {
+	border: $border-width-md solid $border !important;
+	border-radius: $border-radius-lg !important;
+	border-color: transparent !important;
+	background: linear-gradient(90deg, $gradient-text-purple, $gradient-text-blue) border-box !important;
```

Comment:
Should these vars refer to the vivid start and end?

Reply:
There was a brief conversation about trying the existing purple-blue gradient but decided to stick with the vivid colors after I abandoned the PR. Revertid the last commit to bring that back, forgot to push that change up.